src/Pages/yeonseonglee/Login/Login.js

```diff
@@ -0,0 +1,176 @@
+import React, { Component } from "react";
+import { withRouter } from "react-router-dom";
+import "./Login.scss";
+
+const info = [
+  "소개",
+  "도움말",
+  "홍보 센터",
+  "API",
+  "채용 정보",
+  "개인정보처리방침",
+  "약관",
+  "위치",
+  "인기 계정",
+  "해시태그",
+  "언어",
+];
```

Comment:
이렇게 변하지 않는 데이터는 대문자로 적어주세요!
```suggestion
const INFO = [
  "소개",
  "도움말",
  "홍보 센터",
  "API",
  "채용 정보",
  "개인정보처리방침",
  "약관",
  "위치",
  "인기 계정",
  "해시태그",
  "언어",
];
```